zk/src/archive/web/js/zk/widget.js

```diff
@@ -2657,7 +2657,11 @@ function () {
 	 * @return zk.Widget this widget.
 	 * @since 5.0.4
 	 */
-	rerender: function (skipper) {
+	rerender: function (skipper, doLater) {
+		if (doLater) {
+			this.doAfterProcessRerender = {skipper: skipper, doLater: true};

```

Comment:
use arguments but remove doLater

Reply:
we retain the doLater for easy maintenance.